mne/io/snirf/tests/test_snirf.py

```diff
@@ -434,6 +435,23 @@ def test_snirf_kernel_hb():
     assert raw.annotations.description[1] == "StartIti"
 
 
+@requires_testing_data
+@pytest.mark.parametrize(
+    "sfreq,context",
+    (
+        [8.2, nullcontext()],  # sfreq estimated from file is 8.256495
+        [22, pytest.warns(RuntimeWarning, match="User-supplied sampling frequency")],
+    ),
+)
+def test_user_set_sfreq(sfreq, context):
+    """Test reading Kernel SNIRF files with haemoglobin data."""
```

Comment:
oops, copy-paste error

```suggestion
    """Test manually setting sfreq."""
```